plio/io/io_controlnetwork.py

```diff
@@ -29,6 +32,71 @@ def write_filelist(lst, path="fromlist.lis"):
         handle.write('\n')
     return
 
+
+class MessageType(IntEnum):
+    """
+    An enum to mirror the ISIS3 MeasureLogData enum.
+    """
+    GoodnessOfFit = 2
+    MinimumPixelZScore = 3
+    MaximumPixelZScore = 4
+    PixelShift = 5
+    WholePixelCorrelation = 6
+    SubPixelCorrelation = 7 
+
+class Log():
+    
+    def __init__(self, messagetype, value):
+        """
+        A protobuf compliant log object.
+        
+        Parameters
+        ----------
+        messagetype : int or str
+                      Either the integer or string representation from the MessageType enum
+                      
+        value : int or float
+                The value to be stored in the message log
+        """
+        if isinstance(messagetype, int):
+            # by value
+            self.messagetype = MessageType(messagetype)
+        else:
+            # by name
+            print(MessageType[messagetype])
+            self.messagetype = MessageType[messagetype]
+        
+        if not isinstance(value, (float, int)):
+            raise TypeError(f'{value} is not a numeric type')
+        self.value = value
+        
+    def __repr__(self):
+        return f'{self.messagetype.name}: {self.value}'
+        
+    def to_protobuf(self):
+        """
+        Return protobuf compliant measure log object representation
+        of this class.
+        
+        Returns
+        -------
+        log_message : obj
+                      MeasureLogData object suitable to append to a MeasureLog
+                      repeated field.
+        """
+        # I do not see a better way to get to the inner MeasureLogData obj than this
+        # imports were not working because it looks like these need to instantiate off
+        # an object
```

Comment:
Is there a reason this method of doing things is bad/undesirable?

Reply:
It's just a little wasteful to create 3 objects and then only care about 1 of them. The standard way to this would `from ControlPointFileEntryV0002.Measure import MeasureLogData` and then just do `MeasureLogData()` and only create the log data protobuf. This creates a control point protobuf, a control measure protobuf, and the log data protobuf

Reply:
The issue is that the imports in that form are not working because it looks like the Measure has to be instantiated already for the MeasureLogData to be usable/importable/anything. Operator error perhaps?